fjsjsf.js

```diff
@@ -0,0 +1,9 @@
+var myarray = []
+
+const addedNumsArray = myarray.concat();
```

Comment:
addedNumsArray needs to be put into a function. Here again, it looks like the thing referenced in the tests is just a variable